1-select-tasks/15.txt

```diff
@@ -1,3 +1,22 @@
 Create a SQL query to return the total number of orders for every month in 1997 year:
 
-| January | February | March | April | May | June | July | August | September | November | December |
\ No newline at end of file
+| January | February | March | April | May | June | July | August | September | November | December |
+
+--task15
+SELECT 
+	COUNT(CASE WHEN MONTH([o].[OrderDate]) = 1 THEN 1 ELSE NULL END) AS [january],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 2 THEN 1 ELSE NULL END) AS [february],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 3 THEN 1 ELSE NULL END) AS [march],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 4 THEN 1 ELSE NULL END) AS [april],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 5 THEN 1 ELSE NULL END) AS [may],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 6 THEN 1 ELSE NULL END) AS [june],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 7 THEN 1 ELSE NULL END) AS [july],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 8 THEN 1 ELSE NULL END) AS [august],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 9 THEN 1 ELSE NULL END) AS [september],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 10 THEN 1 ELSE NULL END) AS [october],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 11 THEN 1 ELSE NULL END) AS [november],
+    COUNT(CASE WHEN MONTH([o].[OrderDate]) = 12 THEN 1 ELSE NULL END) AS [desember]
+FROM [dbo].[Orders] [o] 
+WHERE YEAR([o].[OrderDate]) = 1997
+
+
```

Comment:
Well done! try to solve this with [pivot function](https://technet.microsoft.com/en-us/library/ms177410(v=sql.105).aspx). Add another file with the second solution